mlu_op.h

```diff
@@ -13253,6 +13253,271 @@ mluOpDCNBackwardData(mluOpHandle_t handle,
                      const mluOpTensorDescriptor_t grad_mask_desc,
                      void *grad_mask);
 
+// Group:FFT
+/*! @brief The descriptor of FFT (Fast Fourier Transform) operation that holds FFT information including
+
+    the tensor descriptor of input tensor and output tensor, the rank of FFT, the FFT size on each
+    dimension, the size of reserved space and the size of workspace.
+    *
+    You need to call the ::mluOpCreateFFTPlan function to create a descriptor for the FFT operation, and call
+    the ::mluOpMakeFFTPlanMany function to set the information of the FFT operation to the descriptor.
+    Then, you need to allocate the reserved space and set the space to the FFT descriptor by ::mluOpSetFFTReserveArea.
+    AT the end you need to destroy the Cambricon MLUOP context with the ::mluOpDestroyFFTPlan.
+    */
```

Comment:
```suggestion
/*! 
 * @brief The descriptor of FFT (Fast Fourier Transform) operation that holds FFT information including
 *
 * the tensor descriptor of input tensor and output tensor, the rank of FFT, the FFT size on each
 * dimension, the size of reserved space and the size of workspace.
 *
 * You need to call the ::mluOpCreateFFTPlan function to create a descriptor for the FFT operation, and call
 * the ::mluOpMakeFFTPlanMany function to set the information of the FFT operation to the descriptor.
 * Then, you need to allocate the reserved space and set the space to the FFT descriptor by ::mluOpSetFFTReserveArea.
 * AT the end you need to destroy the Cambricon MLUOP context with the ::mluOpDestroyFFTPlan.
 */
```
mlu_op.h 中的接口注释需要写成此格式，后面的注释同理